pyquil/gates.py

```diff
@@ -630,7 +650,12 @@ def LOAD(target_reg, region_name, offset_reg):
     return ClassicalLoad(unpack_classical_reg(target_reg), region_name, unpack_classical_reg(offset_reg))
 
 
-def STORE(region_name, offset_reg, source):
+# TODO:(appleby) Docstring wrong here? ClassicalStore's __init__ will throw an exception if either
+# of the last two args are not MemoryReferences. Perhaps the comment should say that offset_reg can
+# be either MemoryReference or constant (i.e. MemoryReferenceDesignator)?
```

Comment:
Comment looks correct here according to quil spec. Possibly a bug in `ClassicalStore`, which should accept a constant or mem ref as the final argument.

Reply:
https://github.com/rigetti/pyquil/issues/815

Reply:
Aha! Fixed now in local branch. Will add a "closes" keyword. thx.